addons/website_sale/views/res_config_settings_views.xml

```diff
@@ -59,6 +59,19 @@
                             </div>
                         </div>
                     </div>
+
+                    <div class="col-12 col-lg-6 o_setting_box" id="cart_redirect_setting">
+                        <div class="o_setting_left_pane">
+                            <field name="cart_add_on_page"/>
+                        </div>
+                        <div class="o_setting_right_pane">
+                            <label for="cart_add_on_page"/>
+                            <div class="text-muted">
+                                No redirect when the user adds a product to page.
```

Comment:
to page... or to cart... ?

MAybe @sbuhl can give feedback on Ux

![image](https://user-images.githubusercontent.com/4848797/103543132-50a93d80-4e9e-11eb-9ae2-80c51660c207.png)


And since this option if website-specific, you need to add the earth ;)

Reply:
`to cart` obviously :-)